package/libpiksi/libpiksi/src/settings.c

```diff
@@ -182,53 +182,56 @@ static int settings_deregister_read_resp_callback(settings_ctx_t *ctx);
 static int float_to_string(const void *priv, char *str, int slen, const void *blob, int blen)
 {
   (void)priv;
-
+  assert(slen >= 0);
```

Comment:
Note these are leftover from previous work on this module, will most likely drop all changes here once libsettings is merged... (also might just disable warnings for this file to get this merged).